Backend/dev/migrations/special-zone/special-zone.sql

```diff
@@ -0,0 +1,19 @@
+CREATE FUNCTION atlas_special_zone.no_overlaps_in_special_zone(id character(36), g geometry(MultiPolygon,4326))
+RETURNS boolean AS $$
+SELECT NOT EXISTS (
```

Comment:
Does this query use spatial index?

Might be fine since adding new zones is very in frequent to reading the zones. Still better to have perf understanding of the query